src/markup/codemirror/create.ts

```diff
@@ -171,7 +183,11 @@ export function createCodemirror(params: CreateCodemirrorParams) {
                             title,
                         },
                     ])(editor);
+
+                    return true;
                 }
+
+                return true;
```

Comment:
We want to keep default paste handling by codemirror if we haven't done anything ourselves. So, should be `return false`